examples/lib/defaultHtmlStepUi.js

```diff
@@ -47,8 +47,9 @@ function DefaultHtmlStepUi(_sequencer, options) {
               <div class="row step">\
                 <div class="col-md-4 details container-fluid">\
                   <div class="cal collapse in"><p>' +
-                    '<a href="https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#' + step.name + '-module">' + (step.description || '') + '</a>' +
-                 '</p></div>\
+                    if(step.name !== 'load-image') {
```

Comment:
Oops. We just want to remove the link. Not the description of the step. You will have to add an `if` condition or ternary operator only for the `href` attribute not the whole link. Or just display a `p` instead of `a`